boards/actinius-icarus/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+authors = [
+   "Jonathan Pallant <jonathan.pallant@42technology.com>",
+]
+categories = ["embedded", "no-std"]
+description = "BSP for the Actinius Icarus"
+keywords = ["arm", "nrf9160"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+name = "actinius-icarus-bsp"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+cortex-m = "0.6.0"
+cortex-m-rt = "0.6.7"
```

Comment:
Could you please set this to 0.6.10? We should do this for all other crates as well, but that can be another PR.

Reply:
Done